File: algo/lc.80.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums)<=2:
            return len(nums)
        
        idx = 2
        for i in range(2, len(nums)):
            if (nums[i] != nums[idx-2]):
                nums[idx] = nums[i]
                idx += 1
        return idx

File: algo/lc.80.cc
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() <= 2) return nums.size();
        
        int idx = 2;
        for (int j = 2; j<nums.size(); j++) {
            if (nums[j] != nums[idx-2]) {
                nums[idx] = nums[j];
                idx++;
            }
        }
        return idx;
    }
};
